src/harness/reference_models/fss_purge/testdata/cbsd_5.json

```diff
@@ -0,0 +1,45 @@
+{
+	"id": "device_5_id",
+	"registrationRequest": {
+					"fccId": "fccId5",
+		            "cbsdCategory": "B",
+		            "callSign": "KPPP",
+		            "userId": "cbsd_5",
+		            "airInterface": {
+    				  "radioTechnology": "E_UTRA",
+                      "supportedSpec": "LTE-Rel10"
+                    },
+		            "cbsdInfo": {
+                      "vendor": "Equipment Maker C"
+                    },
+
+					"cbsdSerialNumber": "serial5",
+                    "measCapability": [ "EUTRA_CARRIER_RSSI_ALWAYS" ],
+                    "installationParam": {
+                      "latitude": 38.98527,
+                      "longitude": -77.45583,
+                      "height": 75,
+                      "heightType": "AGL",
+                      "indoorDeployment": false,
+                      "antennaAzimuth": 170,
+                      "antennaDowntilt": -5,
+                      "antennaGain": 0,
+                      "antennaBeamwidth": 60,
+                      "eirpCapability": 47
+                    }
+			    	},
+    "grantRequests": [{
+					"channelType": "GAA",
+					"grantExpireTime": "calculated at runtime",
+					"id": "cbsd_5_grant_0",
+					"operationParam": {
+						"maxEirp": 37,
+						"operationFrequencyRange": {
+							"highFrequency": 3700000000.0,
+							"lowFrequency": 3690000000.0
+						}
```

Comment:
maxEirp should be with decimal points
Frequency should be integer

Reply:
fixed